.github/dependabot.yml

```diff
@@ -1,24 +1,23 @@
 # Please see the documentation for all configuration options:
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
 version: 2
 updates:
   - package-ecosystem: "npm"
     directory: "/"
     schedule:
       interval: "daily"
-    labels:
-      - "dependabot"
-    # Too much noise with patch releases...
+      # By default, Dependabot checks for new versions at a random set time for the repository.
+      time: "08:00"
+      timezone: "Europe/Berlin"
     ignore:
       - dependency-name: "*"
         update-types: ["version-update:semver-patch"]
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "daily"
-    labels:
```

Comment:
we don't need dependabot label anymore since by default dependabot will add labels